.dotnet/src/Custom/FineTuning/FineTuningJob.cs

```diff
@@ -0,0 +1,23 @@
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJob")]
+public partial class FineTuningJob
+{
+    internal static FineTuningJob FromResponse(PipelineResponse pipelineResponse)
+    {
+        using var document = JsonDocument.Parse(pipelineResponse.Content);
+        return DeserializeFineTuningJob(document.RootElement);
+    }
+
+    public FineTuningJobStatus Status { get; }
+
+    public string Object { get; }

```

Comment:
Can make this private and even a field if desired, e.g. `[CodeGenMember("Object")] private string _object;`